packages/react/src/components/accordion/accordion-item.tsx

```diff
@@ -0,0 +1,166 @@
+    font-size: 0.875rem;
+    font-weight: var(--font-normal);
+    justify-content: start;
+    letter-spacing: 0.015rem;
+    line-height: 1.5;
+    min-height: var(--spacing-5x);
+    padding: var(--spacing-1x);
+    text-align: left;
+    text-transform: none;
+    transition-delay: 0.5s;
+    transition-duration: 0.2s;
+    transition-property: border-radius;
+    transition-timing-function: ease;
+    width: 100%;
+
+    &.expanded {
+        background: ${({ theme }) => theme.greys.white};
+        color: ${({ theme }) => theme.greys['neutral-90']};
+        transition-delay: 0s;
+        transition-duration: 0.1s;
+        transition-property: border-radius;
+        transition-timing-function: ease;
+    }
+
+    &:focus {
+        box-shadow: ${({ theme }) => theme.tokens['focus-box-shadow-inset']};
+        color: ${({ theme }) => theme.greys['neutral-90']};
+    }
+
+    &:hover {
+        background: ${({ theme }) => theme.greys.grey};
+        border-color: ${({ theme }) => theme.greys['neutral-90']};
+        color: ${({ theme }) => theme.greys['neutral-90']};
+    }
+
+    > svg {
+        height: 1rem;
+        margin: 3px var(--spacing-1halfx) 0 3px;
+        width: 1rem;
+    }
+
+    &:disabled {
+        background-color: ${({ theme }) => theme.greys['light-grey']};
+        &:hover {
+            border-color: ${({ theme }) => theme.greys.grey};
+            color: ${({ theme }) => theme.greys['mid-grey']};
+        }
+        > svg {
+            color: ${({ theme }) => theme.greys['mid-grey']};
+        }
+    }
+`;
+
+export const AccordionItem: React.FC<AccordionItemProps> = ({
+    title,
+    id,
+    type,
+    tag,
+    expanded,
+    onToggle,
+    disabled,
+    children,
+    onKeyDown,
+    buttonRef,
+}) => {
+    const headerId = id;
+    const panelId = `panel-${id}`;
+
+    const itemType = type ?? 'medium';
+    const itemTag = tag ?? 'h3';
+    const isExpanded = expanded ?? false;
+
+    return (
+        <>
+            <HeadingStyled type={itemType} tag={itemTag} noMargin>
+                <ButtonStyled
+                    id={headerId}
+                    className={isExpanded ? 'expanded' : ''}
+                    buttonType="tertiary"
+                    label={title}
+                    aria-expanded={isExpanded}
+                    aria-controls={panelId}
+                    onClick={onToggle}
+                    expanded={isExpanded}
+                    disabled={disabled}
+                    onKeyDown={onKeyDown}
+                    ref={buttonRef}
+                >
+                    <Icon name={isExpanded ? 'caretDown' : 'caretRight'} aria-hidden="true" />
+                </ButtonStyled>
+            </HeadingStyled>
+            <AccordionSection
+                className={isExpanded ? 'expanded' : ''}
+                id={panelId}
+                aria-labelledby={headerId}
+                aria-expanded={isExpanded}
+                role="region"
```

Comment:
You probably don't need the `role="region"` since the HTML tag is a `<section>`.

> Using the `<section>` element will automatically communicate that a section has a role of region if it is given an accessible name. If at all possible, prefer using `<section>` instead.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role

Reply:
you are right.